[Remote Cluster] Improve UX

SoniaSanzV opened the pull request:
Closes https://github.com/elastic/kibana/issues/199664
Part of https://github.com/elastic/kibana/issues/205027 (removes .scss file)
## Summary

<details>
  <summary>Screenshots</summary>

  **Add remote cluster in Cloud**

<img width="1512" alt="Screenshot 2025-01-28 at 10 34 23" src="https://github.com/user-attachments/assets/b8cf24be-8e04-4629-939c-c8d781a727e4" />

<img width="1500" alt="Screenshot 2025-01-28 at 10 36 03" src="https://github.com/user-attachments/assets/612dbc29-f5cb-4809-a4e7-95a99cd2643a" />


**Cloud/Certificate**

<img width="1502" alt="Screenshot 2025-01-28 at 10 36 39" src="https://github.com/user-attachments/assets/b174735f-41ce-4fdb-872e-5db26e3400e0" />

**Cloud/API**

<img width="1505" alt="Screenshot 2025-01-28 at 10 36 28" src="https://github.com/user-attachments/assets/c6c03589-1bf8-4e47-8c8d-2ab8da897c4e" />

**Edit remote cluster in Cloud**

<img width="1506" alt="Screenshot 2025-01-28 at 10 37 47" src="https://github.com/user-attachments/assets/bc74e3fb-28e8-41ca-90d9-c3b09c9b8532" />

**Add remote cluster in Stateful/API Key and Cert**

<img width="1501" alt="Screenshot 2025-01-28 at 10 31 36" src="https://github.com/user-attachments/assets/2ed78cc3-2d5a-4090-9b3d-fd6c78b73c00" />
<img width="1503" alt="Screenshot 2025-01-28 at 10 32 02" src="https://github.com/user-attachments/assets/33487816-613e-4e8b-8ff4-dfb73ff4c67c" />
<img width="756" alt="Screenshot 2025-01-28 at 10 32 13" src="https://github.com/user-attachments/assets/e387f6e8-51f0-4b8e-859d-88f5112897f9" />

**Edit remote cluster in Stateful**

<img width="1505" alt="Screenshot 2025-01-28 at 10 31 08" src="https://github.com/user-attachments/assets/e176dd9e-0336-412e-a231-a07f92a6ae0f" />
</details>

### About this PR

#### Typescript
All files in this PR have been created in typescript or moved to typescript. It has been intended that everything follows strict typing but in two cases: `RemoteClusterAdd.container` and `RemoteClusterEdit.container`. In these cases it has been chosen to add type `any()`, which is basically the same as it already existed when they were `.js` files. The reason for not adding strict typing to these files is that it would add more noise to this PR and is something that can be done as a follow-up.

No files were migrated to typescript that did not directly affect the changes within the scope of this PR. 

#### Styles
I removed `_hacks.scss`. The only class that was in use was `remoteClusterSkipIfUnavailableSwitch`, and only the `padding-top` applied. I moved this padding to `emotion` in the `remote_cluster_form` file, using `euiTheme`. 

I realized that the plugin has some `classNames` in other files that doesn't exist in any style file. I'm pretty sure it's dead code and can be removed in a follow-up.

#### Possible follow-ups
* Remove dead css classes
* Add types to `RemoteClusterAdd.container` and `RemoteClusterEdit.container`
* Migrate the rest of the files in the plugin to typescript
* Move RequestFlyout to [ViewApiRequestFlyout](https://github.com/elastic/kibana/blob/cfb1997d7b7d9b64c8f38796186a45b7fe3fcced/x-pack/platform/plugins/shared/ingest_pipelines/public/application/components/pipeline_form/pipeline_request_flyout/pipeline_request_flyout.tsx#L55)
* Add Functional test that actually connects the clusters

### How to test

### Checklist

- [x] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/README.md)
- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [x] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7776

<!--ONMERGE {"backportTargets":["8.x"]} ONMERGE-->

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!

elasticmachine commented:
Pinging @elastic/kibana-management (Team:Kibana Management)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/268029)
* Commit: 9819d94f222e55d6ab699c94c02ed364447d0ebe

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/268029#019473ff-d929-48d2-96a9-76c51f4bf0c3)

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268029","number":268029,"commit":"9819d94f222e55d6ab699c94c02ed364447d0ebe"}],"number":268029}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/268205)
* Commit: b524d1dbbdc44d2e22ce76f1846b749564515ae1

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/268205#01947568-b57e-4bf9-8738-ddfc32ded0ef)

### History
* :broken_heart: [Build #268029](https://buildkite.com/elastic/kibana-pull-request/builds/268029) failed 9819d94f222e55d6ab699c94c02ed364447d0ebe

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268205","number":268205,"commit":"b524d1dbbdc44d2e22ce76f1846b749564515ae1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268029","number":268029,"commit":"9819d94f222e55d6ab699c94c02ed364447d0ebe"}],"number":268205}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/268212)
* Commit: c1e243b649c139804f480987a8463bd7d46fbffc

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/268212#01947584-67b9-4244-b1f7-e82162be6fa5)

### History
* :broken_heart: [Build #268205](https://buildkite.com/elastic/kibana-pull-request/builds/268205) failed b524d1dbbdc44d2e22ce76f1846b749564515ae1
* :broken_heart: [Build #268029](https://buildkite.com/elastic/kibana-pull-request/builds/268029) failed 9819d94f222e55d6ab699c94c02ed364447d0ebe

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268212","number":268212,"commit":"c1e243b649c139804f480987a8463bd7d46fbffc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268205","number":268205,"commit":"b524d1dbbdc44d2e22ce76f1846b749564515ae1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268029","number":268029,"commit":"9819d94f222e55d6ab699c94c02ed364447d0ebe"}],"number":268212}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/268360)
* Commit: 1c38197ba714b481be8ea29c3c393c5721bb77a3
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #268212](https://buildkite.com/elastic/kibana-pull-request/builds/268212) failed c1e243b649c139804f480987a8463bd7d46fbffc
* :broken_heart: [Build #268205](https://buildkite.com/elastic/kibana-pull-request/builds/268205) failed b524d1dbbdc44d2e22ce76f1846b749564515ae1
* :broken_heart: [Build #268029](https://buildkite.com/elastic/kibana-pull-request/builds/268029) failed 9819d94f222e55d6ab699c94c02ed364447d0ebe

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268360","number":268360,"commit":"1c38197ba714b481be8ea29c3c393c5721bb77a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268212","number":268212,"commit":"c1e243b649c139804f480987a8463bd7d46fbffc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268205","number":268205,"commit":"b524d1dbbdc44d2e22ce76f1846b749564515ae1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268029","number":268029,"commit":"9819d94f222e55d6ab699c94c02ed364447d0ebe"}],"number":268360}
buildkite-pr-comment-->

ElenaStoeva reviewed (commented):
Great job @SoniaSanzV! I tested locally and noticed a few things:

- When you first select API keys and go to the second step, but then go back to the first step and select "Certificates", the selection is not saved:

https://github.com/user-attachments/assets/561ad31c-94f5-44be-b0d9-9a5cf03d3c6a

- The help text of the fields in the second step doesn't seem to be updated as in the screenshots in https://github.com/elastic/kibana/issues/199664:

<img width="1159" alt="Screenshot 2025-01-20 at 14 54 01" src="https://github.com/user-attachments/assets/89f77452-42e2-448b-94b5-fa444a741c09" />

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/268529)
* Commit: 3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #268360](https://buildkite.com/elastic/kibana-pull-request/builds/268360) failed 1c38197ba714b481be8ea29c3c393c5721bb77a3
* :broken_heart: [Build #268212](https://buildkite.com/elastic/kibana-pull-request/builds/268212) failed c1e243b649c139804f480987a8463bd7d46fbffc
* :broken_heart: [Build #268205](https://buildkite.com/elastic/kibana-pull-request/builds/268205) failed b524d1dbbdc44d2e22ce76f1846b749564515ae1
* :broken_heart: [Build #268029](https://buildkite.com/elastic/kibana-pull-request/builds/268029) failed 9819d94f222e55d6ab699c94c02ed364447d0ebe

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268529","number":268529,"commit":"3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268360","number":268360,"commit":"1c38197ba714b481be8ea29c3c393c5721bb77a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268212","number":268212,"commit":"c1e243b649c139804f480987a8463bd7d46fbffc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268205","number":268205,"commit":"b524d1dbbdc44d2e22ce76f1846b749564515ae1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268029","number":268029,"commit":"9819d94f222e55d6ab699c94c02ed364447d0ebe"}],"number":268529}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/268732)
* Commit: d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### Failed CI Steps
* [x-pack/test_serverless/functional/test_suites/observability/common_configs/config.group5.ts](https://buildkite.com/elastic/kibana-pull-request/builds/268732#01948887-745f-4941-beec-fa1f2c8fd189)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/268732#01948887-745f-4941-beec-fa1f2c8fd189) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/268732/jobs/01948887-745f-4941-beec-fa1f2c8fd189/artifacts/01948894-16f7-4655-9ab6-a47f38596c6a) x-pack/test_serverless/functional/test_suites/observability/common_configs/config.group5.ts / discover/group1 discover test nested query should support querying on nested fields

### History
* :green_heart: [Build #268529](https://buildkite.com/elastic/kibana-pull-request/builds/268529) succeeded 3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f
* :broken_heart: [Build #268360](https://buildkite.com/elastic/kibana-pull-request/builds/268360) failed 1c38197ba714b481be8ea29c3c393c5721bb77a3
* :broken_heart: [Build #268212](https://buildkite.com/elastic/kibana-pull-request/builds/268212) failed c1e243b649c139804f480987a8463bd7d46fbffc
* :broken_heart: [Build #268205](https://buildkite.com/elastic/kibana-pull-request/builds/268205) failed b524d1dbbdc44d2e22ce76f1846b749564515ae1
* :broken_heart: [Build #268029](https://buildkite.com/elastic/kibana-pull-request/builds/268029) failed 9819d94f222e55d6ab699c94c02ed364447d0ebe

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268732","number":268732,"commit":"d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268529","number":268529,"commit":"3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268360","number":268360,"commit":"1c38197ba714b481be8ea29c3c393c5721bb77a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268212","number":268212,"commit":"c1e243b649c139804f480987a8463bd7d46fbffc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268205","number":268205,"commit":"b524d1dbbdc44d2e22ce76f1846b749564515ae1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268029","number":268029,"commit":"9819d94f222e55d6ab699c94c02ed364447d0ebe"}],"number":268732}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/269141)
* Commit: aa449e85f100d255f927c7bb957f909a4eb9ef5d

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/269141#01948d4c-1934-4546-bae4-49cab233c4a4)

### History
* :yellow_heart: [Build #268732](https://buildkite.com/elastic/kibana-pull-request/builds/268732) was flaky d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe
* :green_heart: [Build #268529](https://buildkite.com/elastic/kibana-pull-request/builds/268529) succeeded 3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f
* :broken_heart: [Build #268360](https://buildkite.com/elastic/kibana-pull-request/builds/268360) failed 1c38197ba714b481be8ea29c3c393c5721bb77a3
* :broken_heart: [Build #268212](https://buildkite.com/elastic/kibana-pull-request/builds/268212) failed c1e243b649c139804f480987a8463bd7d46fbffc
* :broken_heart: [Build #268205](https://buildkite.com/elastic/kibana-pull-request/builds/268205) failed b524d1dbbdc44d2e22ce76f1846b749564515ae1

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269141","number":269141,"commit":"aa449e85f100d255f927c7bb957f909a4eb9ef5d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268732","number":268732,"commit":"d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268529","number":268529,"commit":"3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268360","number":268360,"commit":"1c38197ba714b481be8ea29c3c393c5721bb77a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268212","number":268212,"commit":"c1e243b649c139804f480987a8463bd7d46fbffc"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268205","number":268205,"commit":"b524d1dbbdc44d2e22ce76f1846b749564515ae1"}],"number":269141}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/269158)
* Commit: 52ad69f167e44c4540e894b65fa7447aa6c67b08

### Failed CI Steps
* [Linting (with types)](https://buildkite.com/elastic/kibana-pull-request/builds/269158#01948d74-8434-46b1-93cb-6d7b16c89431)

### History
* :broken_heart: [Build #269141](https://buildkite.com/elastic/kibana-pull-request/builds/269141) failed aa449e85f100d255f927c7bb957f909a4eb9ef5d
* :yellow_heart: [Build #268732](https://buildkite.com/elastic/kibana-pull-request/builds/268732) was flaky d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe
* :green_heart: [Build #268529](https://buildkite.com/elastic/kibana-pull-request/builds/268529) succeeded 3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f
* :broken_heart: [Build #268360](https://buildkite.com/elastic/kibana-pull-request/builds/268360) failed 1c38197ba714b481be8ea29c3c393c5721bb77a3
* :broken_heart: [Build #268212](https://buildkite.com/elastic/kibana-pull-request/builds/268212) failed c1e243b649c139804f480987a8463bd7d46fbffc

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269158","number":269158,"commit":"52ad69f167e44c4540e894b65fa7447aa6c67b08"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269141","number":269141,"commit":"aa449e85f100d255f927c7bb957f909a4eb9ef5d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268732","number":268732,"commit":"d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268529","number":268529,"commit":"3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268360","number":268360,"commit":"1c38197ba714b481be8ea29c3c393c5721bb77a3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268212","number":268212,"commit":"c1e243b649c139804f480987a8463bd7d46fbffc"}],"number":269158}
buildkite-pr-comment-->

sabarasaba reviewed (commented):
Nice job @SoniaSanzV patching up this flow and migrating all the files to ts!

I have some initial feedback in terms of UX regarding the `cancel` button:

![Screenshot 2025-01-22 at 12 01 09](https://github.com/user-attachments/assets/b3631f89-34d2-4843-b082-fc9972dbeb8c)

Do we need this at all? Looking at other UIs having a cancel action is not a pattern we've been using, I think due to having the crumbs, browser back button and left side nav (perhaps more of a question for @jovana-andjelkovic). This might also lead to unexpected loss of data if hit accidentally, for example if you fill your connection information and hit cancel: all information the user entered is lost.

I would suggest replacing it with a button that allows the user to go one step back in the wizard.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/269187)
* Commit: 5f64249308319c9bfdecd4244bd774ac8309ab51
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #269158](https://buildkite.com/elastic/kibana-pull-request/builds/269158) failed 52ad69f167e44c4540e894b65fa7447aa6c67b08
* :broken_heart: [Build #269141](https://buildkite.com/elastic/kibana-pull-request/builds/269141) failed aa449e85f100d255f927c7bb957f909a4eb9ef5d
* :yellow_heart: [Build #268732](https://buildkite.com/elastic/kibana-pull-request/builds/268732) was flaky d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe
* :green_heart: [Build #268529](https://buildkite.com/elastic/kibana-pull-request/builds/268529) succeeded 3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f
* :broken_heart: [Build #268360](https://buildkite.com/elastic/kibana-pull-request/builds/268360) failed 1c38197ba714b481be8ea29c3c393c5721bb77a3

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269187","number":269187,"commit":"5f64249308319c9bfdecd4244bd774ac8309ab51"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269158","number":269158,"commit":"52ad69f167e44c4540e894b65fa7447aa6c67b08"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269141","number":269141,"commit":"aa449e85f100d255f927c7bb957f909a4eb9ef5d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268732","number":268732,"commit":"d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268529","number":268529,"commit":"3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268360","number":268360,"commit":"1c38197ba714b481be8ea29c3c393c5721bb77a3"}],"number":269187}
buildkite-pr-comment-->

sabarasaba commented on x-pack/platform/plugins/private/remote_clusters/public/application/sections/components/remote_cluster_config_steps/remote_cluster_setup_trust/remote_cluster_setup_trust.tsx:
```diff
@@ -48,32 +40,65 @@ const i18nTexts = {
+  const { canUseAPIKeyTrustModel } = useContext(AppContext);
+  const [securityModel, setSecurityModel] = useState<string>(currentSecurityModel);
+
+  const selectModeButton = (securityModelType: string, testSubj: string) => {
+    return securityModel === securityModelType ? (
+      <EuiButton
+        onClick={() => {
+          setSecurityModel(securityModelType);
+          onSecurityChange(securityModelType);
+        }}
+        color="success"
+        iconType="check"
+        iconSide="left"
+        fullWidth
+        data-test-subj={testSubj}
+      >
+        <FormattedMessage
+          id="xpack.remoteClusters.clusterWizard.trustStep.selected"
+          defaultMessage="Selected"
+        />
+      </EuiButton>
+    ) : (
+      <EuiButton
+        onClick={() => {
+          setSecurityModel(securityModelType);
+          onSecurityChange(securityModelType);
+        }}
+        color="text"
+        fullWidth
+        data-test-subj={testSubj}
+      >
+        <FormattedMessage
+          id="xpack.remoteClusters.clusterWizard.trustStep.docs"
+          defaultMessage="Select"
+        />
+      </EuiButton>
+    );
+  };
```
How do you feel about avoiding some of the code repetition with something like?

```suggestion
const selectModeButton = (securityModelType: string, testSubj: string) => {
  const isSelected = securityModel === securityModelType;
  const buttonProps = {
    onClick: () => {
      setSecurityModel(securityModelType);
      onSecurityChange(securityModelType);
    },
    color: isSelected ? 'success': 'text',
    fullWidth: true,
    "data-test-subj": testSubj,
    iconType: isSelected ? 'check': undefined,
    iconSide: isSelected ? 'left': undefined,
  };

  return (
    <EuiButton {...buttonProps}>
      <FormattedMessage
        id={
          isSelected
            ? "xpack.remoteClusters.clusterWizard.trustStep.selected"
            : "xpack.remoteClusters.clusterWizard.trustStep.docs"
        }
        defaultMessage={isSelected ? "Selected" : "Select"}
      />
    </EuiButton>
  );
};
```

SoniaSanzV replied:
I updated the EUI component and this is not needed anymore

sabarasaba reviewed (commented):
Left few initial comments, let me know what you think!

jovana-andjelkovic reviewed (commented):
@SoniaSanzV great work! 

I agree with @sabarasaba regarding "Cancel" that should be back on steps 2 & 3 and step 1 can be without cancel as well. I just checked `create index template flow` as an example, and we can follow the same pattern. We used to have a back button here instead of cancel in 2nd step as well, it just got lost over design iterations 😅 

I've checked cloud version and it looks great! I have only one nitpick – if we could align select buttons in step 1, that would be amazing ✨ 
![image](https://github.com/user-attachments/assets/34482db2-f9ce-465d-a453-2e37cc26c171)

SoniaSanzV commented on x-pack/platform/plugins/private/remote_clusters/public/application/sections/components/remote_cluster_config_steps/components/action_buttons.tsx:
```diff
@@ -0,0 +1,93 @@
+        )}
+      </EuiFlexItem>
+      {isRequestVisible && cluster ? (
+        <RequestFlyout cluster={cluster} close={() => setIsRequestVisible(false)} />
```
I like that, but let's address it in a follow-up

SoniaSanzV commented:
Thank you for the reviews @jovana-andjelkovic @sabarasaba @ElenaStoeva! I think I've addressed all your feedback or added it as a follow-up (I hope I'm not missing anything)!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/269623)
* Commit: 58fb4764284686408c886aea49873402f126c660
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :yellow_heart: [Build #269187](https://buildkite.com/elastic/kibana-pull-request/builds/269187) was flaky 5f64249308319c9bfdecd4244bd774ac8309ab51
* :broken_heart: [Build #269158](https://buildkite.com/elastic/kibana-pull-request/builds/269158) failed 52ad69f167e44c4540e894b65fa7447aa6c67b08
* :broken_heart: [Build #269141](https://buildkite.com/elastic/kibana-pull-request/builds/269141) failed aa449e85f100d255f927c7bb957f909a4eb9ef5d
* :yellow_heart: [Build #268732](https://buildkite.com/elastic/kibana-pull-request/builds/268732) was flaky d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe
* :green_heart: [Build #268529](https://buildkite.com/elastic/kibana-pull-request/builds/268529) succeeded 3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269623","number":269623,"commit":"58fb4764284686408c886aea49873402f126c660"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269187","number":269187,"commit":"5f64249308319c9bfdecd4244bd774ac8309ab51"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269158","number":269158,"commit":"52ad69f167e44c4540e894b65fa7447aa6c67b08"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269141","number":269141,"commit":"aa449e85f100d255f927c7bb957f909a4eb9ef5d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268732","number":268732,"commit":"d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268529","number":268529,"commit":"3d2f9e1fe27feefaa5d40abcc4a7d90c99f6d07f"}],"number":269623}
buildkite-pr-comment-->

kibanamachine commented:
## Flaky Test Runner Stats
### 🎉 All tests passed! - [kibana-flaky-test-suite-runner#7776](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7776)
[✅] x-pack/test/functional/apps/remote_clusters/config.ts: 200/200 tests passed.

[see run history](https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds?branch=refs%2Fpull%2F206958%2Fhead)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/269943)
* Commit: 24a4ba68d4cee825ac87cb45ab238ddd678256d1
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :yellow_heart: [Build #269623](https://buildkite.com/elastic/kibana-pull-request/builds/269623) was flaky 58fb4764284686408c886aea49873402f126c660
* :yellow_heart: [Build #269187](https://buildkite.com/elastic/kibana-pull-request/builds/269187) was flaky 5f64249308319c9bfdecd4244bd774ac8309ab51
* :broken_heart: [Build #269158](https://buildkite.com/elastic/kibana-pull-request/builds/269158) failed 52ad69f167e44c4540e894b65fa7447aa6c67b08
* :broken_heart: [Build #269141](https://buildkite.com/elastic/kibana-pull-request/builds/269141) failed aa449e85f100d255f927c7bb957f909a4eb9ef5d
* :yellow_heart: [Build #268732](https://buildkite.com/elastic/kibana-pull-request/builds/268732) was flaky d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269943","number":269943,"commit":"24a4ba68d4cee825ac87cb45ab238ddd678256d1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269623","number":269623,"commit":"58fb4764284686408c886aea49873402f126c660"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269187","number":269187,"commit":"5f64249308319c9bfdecd4244bd774ac8309ab51"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269158","number":269158,"commit":"52ad69f167e44c4540e894b65fa7447aa6c67b08"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269141","number":269141,"commit":"aa449e85f100d255f927c7bb957f909a4eb9ef5d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/268732","number":268732,"commit":"d7b361d344ded2c2fd18d32dcf3ee55c18ccb7fe"}],"number":269943}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/270002)
* Commit: 065cc3b0dffc3c1844f41f5edcc9f952aa54437f
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #269943](https://buildkite.com/elastic/kibana-pull-request/builds/269943) failed 24a4ba68d4cee825ac87cb45ab238ddd678256d1
* :yellow_heart: [Build #269623](https://buildkite.com/elastic/kibana-pull-request/builds/269623) was flaky 58fb4764284686408c886aea49873402f126c660
* :yellow_heart: [Build #269187](https://buildkite.com/elastic/kibana-pull-request/builds/269187) was flaky 5f64249308319c9bfdecd4244bd774ac8309ab51
* :broken_heart: [Build #269158](https://buildkite.com/elastic/kibana-pull-request/builds/269158) failed 52ad69f167e44c4540e894b65fa7447aa6c67b08
* :broken_heart: [Build #269141](https://buildkite.com/elastic/kibana-pull-request/builds/269141) failed aa449e85f100d255f927c7bb957f909a4eb9ef5d

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270002","number":270002,"commit":"065cc3b0dffc3c1844f41f5edcc9f952aa54437f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269943","number":269943,"commit":"24a4ba68d4cee825ac87cb45ab238ddd678256d1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269623","number":269623,"commit":"58fb4764284686408c886aea49873402f126c660"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269187","number":269187,"commit":"5f64249308319c9bfdecd4244bd774ac8309ab51"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269158","number":269158,"commit":"52ad69f167e44c4540e894b65fa7447aa6c67b08"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269141","number":269141,"commit":"aa449e85f100d255f927c7bb957f909a4eb9ef5d"}],"number":270002}
buildkite-pr-comment-->

sabarasaba reviewed (approved):
Latest code changes lgtm, tested locally. Awesome work @SoniaSanzV!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/270816)
* Commit: 69cb0d594652dcc06dbbe1e39c63d58adbaf8c55
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :green_heart: [Build #270002](https://buildkite.com/elastic/kibana-pull-request/builds/270002) succeeded 065cc3b0dffc3c1844f41f5edcc9f952aa54437f
* :broken_heart: [Build #269943](https://buildkite.com/elastic/kibana-pull-request/builds/269943) failed 24a4ba68d4cee825ac87cb45ab238ddd678256d1
* :yellow_heart: [Build #269623](https://buildkite.com/elastic/kibana-pull-request/builds/269623) was flaky 58fb4764284686408c886aea49873402f126c660
* :yellow_heart: [Build #269187](https://buildkite.com/elastic/kibana-pull-request/builds/269187) was flaky 5f64249308319c9bfdecd4244bd774ac8309ab51
* :broken_heart: [Build #269158](https://buildkite.com/elastic/kibana-pull-request/builds/269158) failed 52ad69f167e44c4540e894b65fa7447aa6c67b08

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270816","number":270816,"commit":"69cb0d594652dcc06dbbe1e39c63d58adbaf8c55"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270002","number":270002,"commit":"065cc3b0dffc3c1844f41f5edcc9f952aa54437f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269943","number":269943,"commit":"24a4ba68d4cee825ac87cb45ab238ddd678256d1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269623","number":269623,"commit":"58fb4764284686408c886aea49873402f126c660"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269187","number":269187,"commit":"5f64249308319c9bfdecd4244bd774ac8309ab51"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269158","number":269158,"commit":"52ad69f167e44c4540e894b65fa7447aa6c67b08"}],"number":270816}
buildkite-pr-comment-->

SoniaSanzV commented:
In the last commit:
* Update review step to add last changes (see updated screenshots in issue)
* Adjust max width of the form to match mocks

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/270846)
* Commit: 679d8d1f8b8b335cfc79250d2e699ef4c31219f3
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :green_heart: [Build #270816](https://buildkite.com/elastic/kibana-pull-request/builds/270816) succeeded 69cb0d594652dcc06dbbe1e39c63d58adbaf8c55
* :green_heart: [Build #270002](https://buildkite.com/elastic/kibana-pull-request/builds/270002) succeeded 065cc3b0dffc3c1844f41f5edcc9f952aa54437f
* :broken_heart: [Build #269943](https://buildkite.com/elastic/kibana-pull-request/builds/269943) failed 24a4ba68d4cee825ac87cb45ab238ddd678256d1
* :yellow_heart: [Build #269623](https://buildkite.com/elastic/kibana-pull-request/builds/269623) was flaky 58fb4764284686408c886aea49873402f126c660
* :yellow_heart: [Build #269187](https://buildkite.com/elastic/kibana-pull-request/builds/269187) was flaky 5f64249308319c9bfdecd4244bd774ac8309ab51
* :broken_heart: [Build #269158](https://buildkite.com/elastic/kibana-pull-request/builds/269158) failed 52ad69f167e44c4540e894b65fa7447aa6c67b08

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270846","number":270846,"commit":"679d8d1f8b8b335cfc79250d2e699ef4c31219f3"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270816","number":270816,"commit":"69cb0d594652dcc06dbbe1e39c63d58adbaf8c55"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270002","number":270002,"commit":"065cc3b0dffc3c1844f41f5edcc9f952aa54437f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269943","number":269943,"commit":"24a4ba68d4cee825ac87cb45ab238ddd678256d1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269623","number":269623,"commit":"58fb4764284686408c886aea49873402f126c660"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269187","number":269187,"commit":"5f64249308319c9bfdecd4244bd774ac8309ab51"}],"number":270846}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/270915)
* Commit: 9987e2a918f69da16d7f13c80096d548ba1a3975
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #270846](https://buildkite.com/elastic/kibana-pull-request/builds/270846) failed 679d8d1f8b8b335cfc79250d2e699ef4c31219f3
* :green_heart: [Build #270816](https://buildkite.com/elastic/kibana-pull-request/builds/270816) succeeded 69cb0d594652dcc06dbbe1e39c63d58adbaf8c55
* :green_heart: [Build #270002](https://buildkite.com/elastic/kibana-pull-request/builds/270002) succeeded 065cc3b0dffc3c1844f41f5edcc9f952aa54437f
* :broken_heart: [Build #269943](https://buildkite.com/elastic/kibana-pull-request/builds/269943) failed 24a4ba68d4cee825ac87cb45ab238ddd678256d1
* :yellow_heart: [Build #269623](https://buildkite.com/elastic/kibana-pull-request/builds/269623) was flaky 58fb4764284686408c886aea49873402f126c660

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270915","number":270915,"commit":"9987e2a918f69da16d7f13c80096d548ba1a3975"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270846","number":270846,"commit":"679d8d1f8b8b335cfc79250d2e699ef4c31219f3"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270816","number":270816,"commit":"69cb0d594652dcc06dbbe1e39c63d58adbaf8c55"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270002","number":270002,"commit":"065cc3b0dffc3c1844f41f5edcc9f952aa54437f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269943","number":269943,"commit":"24a4ba68d4cee825ac87cb45ab238ddd678256d1"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269623","number":269623,"commit":"58fb4764284686408c886aea49873402f126c660"}],"number":270915}
buildkite-pr-comment-->

jovana-andjelkovic reviewed (approved):
Looks good to me! Thank you ✨

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/271002)
* Commit: 05f3e6af9801993d06a74bfe55b04ccb6bbcb367
* [Cloud Deployment](https://kibana-pr-206958.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #270915](https://buildkite.com/elastic/kibana-pull-request/builds/270915) failed 9987e2a918f69da16d7f13c80096d548ba1a3975
* :broken_heart: [Build #270846](https://buildkite.com/elastic/kibana-pull-request/builds/270846) failed 679d8d1f8b8b335cfc79250d2e699ef4c31219f3
* :green_heart: [Build #270816](https://buildkite.com/elastic/kibana-pull-request/builds/270816) succeeded 69cb0d594652dcc06dbbe1e39c63d58adbaf8c55
* :green_heart: [Build #270002](https://buildkite.com/elastic/kibana-pull-request/builds/270002) succeeded 065cc3b0dffc3c1844f41f5edcc9f952aa54437f
* :broken_heart: [Build #269943](https://buildkite.com/elastic/kibana-pull-request/builds/269943) failed 24a4ba68d4cee825ac87cb45ab238ddd678256d1

cc @SoniaSanzV

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/271002","number":271002,"commit":"05f3e6af9801993d06a74bfe55b04ccb6bbcb367"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270915","number":270915,"commit":"9987e2a918f69da16d7f13c80096d548ba1a3975"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270846","number":270846,"commit":"679d8d1f8b8b335cfc79250d2e699ef4c31219f3"},{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270816","number":270816,"commit":"69cb0d594652dcc06dbbe1e39c63d58adbaf8c55"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270002","number":270002,"commit":"065cc3b0dffc3c1844f41f5edcc9f952aa54437f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/269943","number":269943,"commit":"24a4ba68d4cee825ac87cb45ab238ddd678256d1"}],"number":271002}
buildkite-pr-comment-->

kibanamachine commented:
Starting backport for target branches: 8.x

https://github.com/elastic/kibana/actions/runs/13015501744

kibanamachine commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/208584">](https://github.com/elastic/kibana/pull/208584)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

<!--- Backport version: 9.4.3 -->
